### PR TITLE
BREAKING CHANGE: Rename `message` prop to `validationText` in Form Fields #DS-676

### PR DESCRIPTION
**Pull request by crishpeen**
And make it dependent on `validationState`.

Sorry for the huge PR.

### CODE REVIEW

File: packages/web-twig/src/Resources/components/Field/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# ValidationText
```
Comment (literat) [inline packages/web-twig/src/Resources/components/Field/README.md:1]: Not sure about this main heading. Maybe `Field` will be better based on the field structure and mention `ValidationText` as subcomponents.

File: packages/web/src/scss/components/CheckboxField/README.md [hunk 2]
```diff
@@ -3,106 +3,92 @@
...
-      <li>Message</li>
+    <span class="CheckboxField__label">Checkbox Label</span>
+    <ul class="CheckboxField__validationText">
+      <li>First Validation text</li>
+      <li>Second Validation text</li>
     </ul>
   </span>
 </label>
-```
+````
```
Comment (literat) [inline packages/web/src/scss/components/CheckboxField/README.md:49]: "````" -> "```"

File: packages/web/src/scss/components/CheckboxField/README.md [hunk 3]
```diff
@@ -3,106 +3,92 @@
...
   </span>
 </label>
 ```
 
-## Additional message:
+## Validation State with Validation Text:
 
+See Validation state [dictionary][dictionary-validation].
+
+````html
```
Comment (literat) [inline packages/web/src/scss/components/CheckboxField/README.md:29]: Duplication of "````html"

File: packages/web/src/scss/components/RadioField/README.md [hunk 4]
```diff
@@ -3,74 +3,70 @@
...
 </label>
 ```
 
 ## Selected state:
 
 ```html
-<label for="radiofield1" class="RadioField">
-  <input type="radio" id="radiofield1" name="example" class="RadioField__input" checked />
+<label for="radiofieldDefaultChecked" class="RadioField">
+  <input type="radio" id="radiofieldDefaultChecked" class="RadioField__input" name="default" checked />
```
Comment (literat) [inline packages/web/src/scss/components/RadioField/README.md:18]: Do we want to have the same `name` in multiple examples?

File: packages/web/src/scss/components/RadioField/README.md [hunk 5]
```diff
@@ -3,74 +3,70 @@
...
 </label>
 ```
 
 ## Disabled state:
 
 ```html
-<label for="radiofield2_0" class="RadioField RadioField--disabled">
-  <input type="radio" id="radiofield2_0" name="example" class="RadioField__input" disabled />
+<label for="radiofieldDisabled" class="RadioField RadioField--disabled">
+  <input type="radio" id="radiofieldDisabled" class="RadioField__input" disabled />
```
Comment (literat) [inline packages/web/src/scss/components/RadioField/README.md:29]: Missing `name`?

**Review by literat — CHANGES_REQUESTED** [review]

**Comment by literat** [comment]
I have splitted this PR by commit/technology into three separate PRs, see https://github.com/lmc-eu/spirit-design-system/pull/899, https://github.com/lmc-eu/spirit-design-system/pull/900, https://github.com/lmc-eu/spirit-design-system/pull/901

**Comment by adamkudrna** [comment]
Closing in favor of the individual PRs.

**Pull request closed without merging**